File: PHP/php_study_3/page9/food.php
<?php 
require_once('menu.php');

class Food extends Menu {
  // Declare a private property named $spiciness
  private $spiciness;
  
  // Override the __construct method of the parent class
  public function __construct($name, $price, $image, $spiciness) {
    parent::__construct($name, $price, $image);
    $this->spiciness = $spiciness;
  }
  
  // Define the getSpiciness method
  public function getSpiciness() {
    return $this->spiciness;
  }
  
}

?>

File: Ruby On Rails/ruby_study_5/page7/food.rb
require "./menu"

class Food < Menu
  attr_accessor :calorie
  
  # Define the info method
  def info
    return "#{self.name} $#{self.price} (#{self.calorie }kcal)"
  end

  def calorie_info
    return "#{self.name} is #{self.calorie}kcal"
  end
end
